mozci/console/commands/push.py

```diff
@@ -350,6 +359,18 @@ def handle(self) -> None:
                     if not tasks_to_retrigger:
                         self.line("No build task detected as potential regression")
                     else:
+                        max_retrigger = self.option("maximum_build_tasks_to_retrigger")
+                        if max_retrigger > 0:
```

Comment:
`0` would be a valid value to disable the feature. Should the check be for `> -1`?

Reply:
On the other hand, if were no retriggers supposed to get scheduled, a warning is not needed if the user is aware.